tests/test_execution.py

```diff
@@ -10,14 +10,17 @@
 
 from http_prompt.context import Context
 from http_prompt.execution import execute
+from http_prompt.xdg import get_tmp_dir
```

Comment:
Python already has a [tempfile](https://docs.python.org/3.5/library/tempfile.html) module that managing temporary files and directories. So writing a `get_tmp_dir` in `http_prompt.xdg` is not really necessary.